ui/history_entries.go

```diff
@@ -25,7 +25,7 @@ func (h *handler) showHistoryPage(w http.ResponseWriter, r *http.Request) {
 	offset := request.QueryIntParam(r, "offset", 0)
 	builder := h.store.NewEntryQueryBuilder(user.ID)
 	builder.WithStatus(model.EntryStatusRead)
-	builder.WithOrder(model.DefaultSortingOrder)
+	builder.WithOrder("changed_at")
 	builder.WithDirection(user.EntryDirection)
```

Comment:
What about the direction? The default value is `ASC`.

Reply:
Yeah, I didn't notice that. Fixed. As you might guess, I use "recent entries first". :)